lua/buftabline/init.lua

```diff
@@ -3,33 +3,54 @@ local u = require("buftabline.utils")
 
 local M = {}
 
+function _G.buftabline()
```

Comment:
Can we use `luaeval` to avoid having to define a global variable? This is how we did it before:

```lua
vim.o.tabline = [[%!luaeval('require("buftabline").build()')]]
```

Reply:
Works for me. I'm still learning the ropes of nvim's lua integration. It's pretty incredible but there's lots there!